scripts/jenkins/cloud/ansible/setup-ssh-access.yml

```diff
@@ -56,6 +56,12 @@
             name: "{{ cloud_env }}"
             ansible_host: "{{ is_physical_deploy | ternary(omit, admin_floating_ip) }}"
             ansible_password: "linux"
+
+        - name: Remove local ssh host key for admin floating IP
+          shell: ssh-keygen -R "{{ admin_floating_ip }}"
```

Comment:
any reason for not using the `known_hosts` module?

https://docs.ansible.com/ansible/2.7/modules/known_hosts_module.html

Reply:
Yes, my ignorance :wink: